config/environments/test.rb

```diff
@@ -30,6 +30,10 @@
   # The :test delivery method accumulates sent emails in the
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
+  config.action_mailer.default_url_options = {
+      host: 'localhost',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.